pkgs/games/solicurses/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, ncurses
+}:
+
+stdenv.mkDerivation rec {
+  pname = "solicurses";
+  version = "unstable-2020-02-13";
+
+  src = fetchFromGitHub {
+    owner = "KaylaPP";
+    repo = "SoliCurses";
+    rev = "dc89ca00fc1711dc449d0a594a4727af22fc35a0";
+    sha256 = "sha256-zWYXpvEnViT/8gsdMU9Ymi4Hw+nwkG6FT/3h5sNMCE4=";
+  };
+
+  buildInputs = [
+    ncurses
+  ];
+
+  preBuild = ''
+    cd build
+  '';
+  installPhase = ''
```

Put a blank line here.

```suggestion

  installPhase = ''
```